x-pack/plugins/apm/server/lib/service_map/get_service_anomalies.ts

```diff
@@ -83,19 +67,25 @@ export async function getServiceAnomalies({
       },
       aggs: {
         services: {
-          terms: { field: 'partition_field_value' },
+          composite: {
+            size: 5000,
```

Comment:
is this just an arbitrary large size?

Reply:
yes, this actually only returned data for 10 services I think?